parsley/message_types.py

```diff
@@ -161,11 +163,32 @@
 }
 
 actuator_id = {
-    'ACTUATOR_VENT_VALVE':      0x00,
-    'ACTUATOR_INJECTOR_VALVE':  0x01,
-    'ACTUATOR_PAYLOAD':         0x02,
-    'ACTUATOR_CAMERAS':         0x03,
-    'ACTUATOR_CANBUS':          0x04,
-    'ACTUATOR_CHARGE':          0x05,
-    'ACTUATOR_RADIO':           0x06
+    'ACTUATOR_VENT_VALVE':       0x00,
+    'ACTUATOR_INJECTOR_VALVE':   0x01,
+    'ACTUATOR_CAMERA_1':         0x02,
+    'ACTUATOR_CAMERA_2':         0x03,
+    'ACTUATOR_CANBUS':           0x04,
+    'ACTUATOR_CHARGE':           0x05,
+    'ACTUATOR_RADIO':            0x06,
+    'ACTUATOR_PAYLOAD_SERVO':    0x07,
+    'ACTUATOR_AIRBRAKES_SERVO':  0x08,
+    'ACTUATOR_AIRBRAKES_ENABLE': 0x09
+}
+
+state_id = {
+    'STATE_POS_X':       0x00,
```

Comment:
I have to fix the enum in canlib to start explicitly at 0. but this is correct on the parsley side